Fixes minor issues on the Getting Started page (#80)
NUbots/NUbook: src/book/03-guides/01-main-codebase/01-getting-started.mdx

   Note that you will need to do `configure` without `-i` the very first time you do this. Using `-i` will show a menu that will allow you to choose the roles to build. Enable the roles you want to build and disable the ones you don't by highlighting them using using the <kbd>↑</kbd> and <kbd>↓</kbd> arrow keys and pressing the spacebar to toggle on and off.

>    If you are following the keyboardwalkfake example, you will need to make sure `keyboardwalkfake` is set to `on`. Find the list of items that are prefixed with `ROLE_` and set all to `off` except for `keyboardwalkfake`, which should be `on`. You do not need to change any other build options in this example.

Reviewer: ```suggestion
   If you are following the `keyboardwalkfake` example, you will need to make sure `keyboardwalkfake` is set to `on`. Find the list of items that are prefixed with `ROLE_` and set all to `off` except for `keyboardwalkfake`, which should be `on`. You do not need to change any other build options in this example.
```

>    If you are following the `keyboardwalkfake` example, you will need to make sure `keyboardwalkfake` is set to `on`. Find the list of items that are prefixed with `ROLE_` and set all to `off` except for `keyboardwalkfake`, which should be `on`. You do not need to change any other build options in this example.

Reviewer 2: Cmake capitalises on and off I'm pretty sure
```suggestion
   If you are following the `keyboardwalkfake` example, you will need to make sure `keyboardwalkfake` is set to `ON`. Find the list of items that are prefixed with `ROLE_` and set all to `OFF` except for `keyboardwalkfake`, which should be `ON`. You do not need to change any other build options in this example.
```

>    Press <kbd>c</kbd> to configure and <kbd>e</kbd> to exit back to the interface. Press <kbd>g</kbd> to generate the build list and exit the menu with <kbd>e</kbd>.

Reviewer 3: ```suggestion
   Press <kbd>c</kbd> to configure. This will show a confirmation menu, press <kbd>e</kbd> to go back to the main menu. Press <kbd>g</kbd> to generate the build list, and then press <kbd>e</kbd> to exit.
```
